src/dialogs/voice-command-dialog/ha-voice-command-dialog.ts

```diff
@@ -132,36 +158,48 @@ export class HaVoiceCommandDialog extends LitElement {
               `
             : ""}
         </paper-dialog-scrollable>
-        <paper-input
-          @keyup=${this._handleKeyUp}
-          label="${this.hass!.localize(
-            `ui.dialogs.voice_command.${
-              SpeechRecognition ? "label_voice" : "label"
-            }`
-          )}"
-          autofocus
-        >
-          ${SpeechRecognition
+        <div class="input">
+          <paper-input
+            @keyup=${this._handleKeyUp}
+            label="${this.hass!.localize(
+              `ui.dialogs.voice_command.${
+                SpeechRecognition ? "label_voice" : "label"
```

Comment:
We shouldn't mutate `window`. Importing a page should never have side effects. This will also make it painful to use it in other pages. We should have a file that just exports them `export const SpeechRecognition = window.SpeechRecognition || window.webkitSpeechRecognition`